features/step_definitions/list_of_venues_steps.rb

```diff
@@ -0,0 +1,34 @@
+Given(/^I live at "(.*?)" address$/) do |address|
+  @address = Venues::Address.from_str(address)
+end
+
+Given(/^a venue with the name "(.*?)" exists$/) do |venue_name|
+  Venues::Creator.execute(
+    venue_repository: Venue,
+    venue_name: venue_name
+  )
+end
+
+Given(/^the venue with the name "(.*?)" has the following branches:$/) do |venue_name, branches|
+  venue = Venues::Finder.find(repository: Venue, term: venue_name)
+  branches.hashes.each do |branch_attrs|
+    Venues::AddBranch.execute(venue: venue, branch_attrs: branch_attrs)
+  end
+end
+
+Given(/^there are no venues near "(.*?)"$/) do |address|
```

Comment:
Unused block argument - `address`. You can omit the argument if you don't care about it.